site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -432,6 +447,15 @@ Indicates that users without a verified email address will be permanently delete
 [field]#{base_field_name}.userDeletePolicy.unverified.numberOfDaysToRetain# [type]#[Integer]# [since]#Available since 1.13.0#::
 The number of days from creation users will be retained before being deleted for not completing email verification. Value must be greater than 0.
 
+[field]#tenant.usernameConfiguration.unique.enabled# [type]#[Boolean]# [since]#Available since 1.27.0#::
+When `true`, FusionAuth will handle username collisions by generating a random suffix. Users with colliding usernames will be able to use the same one, but in the admin and API interfaces, a unique username will be generated.
```

Comment:
" but in the admin and API interfaces, a unique username will be generated." => " but in the admin and API interfaces, a unique username will be displayed."